models/comments.models.js

```diff
@@ -4,7 +4,12 @@ exports.selectCommentsByArticleId = (article_id) => {
     return db.query(`
     SELECT *
     FROM comments
-    WHERE article_id=$1`, [article_id]).then(({rows})=>{
+    WHERE article_id=$1`, [article_id]).then(({ rows }) => {
+        if (rows.length === 0) {
```

Comment:
if there is an existing article that has no comments, the length will be 0 but this isn't a bad thing - it's just that the article hasn't been commented on yet so we don't want to send a 404 here; so 2 things to think about: what should we send in this scenario, and how could we instead deal with an article id that doesnt exist?